docs/features/dev-card.md

```diff
@@ -1,14 +1,19 @@
 ---
 id: dev-card
-title: 'Showcase your open source stats with your dev card'
-sidebar_label: 'Dev Card'
+title: 'Showcase Your Open Source Stats with Your Dev Card'
+sidebar_label: Dev Card
 keywords:
-- dev-card 
+  - dev-card
 ---
 
-The Dev card feature is what we like to call your "open source business card." On the front, you see your username, the number of pull requests you have created, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate. When you click on the back of your card, you see additional information, including your OpenSauced profile's headline, the date of your first contribution on GitHub, and your [PR velocity rate](../../docs/community/pr-velocity.md).
-![screenshot of front and back of a dev card](../../static/img/a-dev-card.png)
+The Dev Card feature is what we like to call your "open source business card."
 
-Whether you are a person who's just beginning their open source journey or a maintainer looking for more contributors and/or sponsors, the Dev Card is a shareable snapshot of your contributions, skills, and engagement in the open source community.
+On the front of your card, you see your profile picture, username, the number of pull requests you have created, the number of repositories you contributed to, and a graph icon that describes your activity rate.
```

Comment:
We might want to be more detail with:

- the number of PRs: is it the total PR in the last week or month?
- the number of repos: is it the total repos since we joined GitHub?